examples/demo/modules/blog/handler/blog_handler.e

```diff
@@ -41,60 +39,63 @@ create
 feature -- execute
 
 	execute (req: WSF_REQUEST; res: WSF_RESPONSE)
-			-- Execute request handler
+			-- Execute request handler for any kind of mapping.
 		do
 			execute_methods (req, res)
 		end
 
 	uri_execute (req: WSF_REQUEST; res: WSF_RESPONSE)
-			-- Execute request handler
+			-- Execute request handler for URI mapping.
 		do
 			execute (req, res)
 		end
 
 	uri_template_execute (req: WSF_REQUEST; res: WSF_RESPONSE)
-			-- Execute request handler
+			-- Execute request handler for URI-template mapping.
 		do
 			execute (req, res)
 		end
 
 feature -- Global Variables
+
 	page_number: NATURAL_32
+			-- Current page number.
 
 feature -- HTTP Methods	
+
 	do_get (req: WSF_REQUEST; res: WSF_RESPONSE)
 			-- <Precursor>
 		local
 			l_page: CMS_RESPONSE
 		do
-
-			-- Read the page number from get variable
+				-- Read page number from path parameter.
 			page_number := page_number_path_parameter (req)
 
-			-- execute response by setting the main content
+				-- Responding with `main_content_html (l_page)'.
 			create {GENERIC_VIEW_CMS_RESPONSE} l_page.make (req, res, api)
-			l_page.set_main_content (main_content_html(l_page))
+			l_page.set_main_content (main_content_html (l_page))
 			l_page.execute
 		end
 
 feature -- Query
 
-
-	posts : LIST[CMS_NODE]
-			-- The posts to list on the given page ordered by date (descending)
+	posts: LIST [CMS_NODE]
+			-- Blog posts to display on given page ordered by date (descending).
 		do
-			Result := node_api.blogs_order_created_desc_limited (entries_per_page, (page_number-1) * entries_per_page)
+			Result := blog_api.blogs_order_created_desc_limited (
+						entries_per_page,
+						entries_per_page * (page_number - 1)
+					)
 		end
 
-	more_than_one_page : BOOLEAN
-			-- Checks if all posts fit on one page (FALSE) or if more than one page is needed (TRUE)
+	multiple_pages_needed : BOOLEAN
+			-- Return if more that one page is needed to display posts.
 		do
 			Result := entries_per_page < total_entries
 		end
 
-
-	pages : NATURAL_32
-			-- Returns the number of pages needed to display all posts
+	pages_count: NATURAL_32
+			-- Number of pages needed to display all posts.
```

Comment:
in Eiffel, we try to get very synthetic comment
avoiding useless preposition such as "a" or "the" when not needed
no need to precise a function returns something, except if we don't find alternative nice comment.
so here, we describe what is the result ... which is the number of pages ...